__docs__/wonder-blocks-icon-button/icon-button.stories.tsx

```diff
@@ -167,14 +168,57 @@ export const Variants: StoryComponentType = {
 };
 
 /**
- * An IconButton on a dark background.
- * Only the primary kind is allowed to have the `light` prop set to true.
+ * IconButton has a `color` that is either `default` (the default, as shown
+ * above) or `destructive` (as can seen below):
```

Comment:
How do you feel about adding a sentence like "Note that the focus ring has the color even when the `kind` is `secondary` or `tertiary`"?

Reply:
ah yeah good point, I'll add that!

Reply:
hmmm giving a second thought, the color changes based on different factors (whether is light, whether is using the khanmigo theme), so I prefer to leave it as it is.